lib/ex_doc/formatter/html.ex

```diff
@@ -48,6 +48,8 @@ defmodule ExDoc.Formatter.HTML do
   defp generate_sidebar_items(modules, exceptions, protocols, output) do
     input = for node <- [%{id: "modules", value: modules}, %{id: "exceptions", value: exceptions}, %{id: "protocols", value: protocols}], !Enum.empty?(node.value), do: node
     content = Templates.sidebar_items_template(input)
+    # Safely reduce the size of the JS code
+    content = Regex.replace(~r/(\s|\n|\h|\v)/, content, "")
```

Comment:
Sure, those are all fine too!

-- 


*José Valimwww.plataformatec.com.br
<http://www.plataformatec.com.br/>Founder and Director of R&D*